apps/console-lite/src/app/components/input-setter/input-setter.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import { t } from '@vegaprotocol/react-helpers';
+import { Button, Input } from '@vegaprotocol/ui-toolkit';
+import type { InputProps } from '@vegaprotocol/ui-toolkit';
+
+interface InputSetterProps {
+  buttonLabel?: string;
+  value: string | number;
+  isInputToggled?: boolean;
```

Comment:
```suggestion
  isInputVisible?: boolean;
```

suggestion (non-blocking): toggle is a very abstract concept which doesn't tell me anything what this is toggled into when i want to set this as the consumer of this component, i need to go in and check the code. i know you need to rename the internal state if you change this, but the internal state matters a lot less compared to the component's interface imho

Reply:
Done